easy-example-experiment.py

```diff
@@ -1,46 +1,37 @@
 #!/usr/bin/env python3
 from pathlib import Path
 from typing import Any, Dict
-
 import numpy as np
 
 from timeeval import TimeEval, DatasetManager, DefaultMetrics, Algorithm, TrainingType, InputDimensionality
-from timeeval.adapters import DockerAdapter, FunctionAdapter
-from timeeval.algorithms import cof
+from timeeval.adapters import DockerAdapter
+from timeeval.adapters import FunctionAdapter # for defining customized algorithm
+from timeeval.algorithms import cof 
 from timeeval.data_types import AlgorithmParameter
 from timeeval.params import FixedParameters
 
-
 def your_algorithm_function(data: AlgorithmParameter, args: Dict[str, Any]) -> np.ndarray:
     if isinstance(data, np.ndarray):
         return np.zeros_like(data)
-    else:  # isinstance(data, pathlib.Path)
```

Comment:
why did you remove this?

Reply:
it was commented code line, not in use. 

I thought it was unnecessary and not useful for first time user

Reply:
I documents the type of the `else`-branch. Please add it back in.